dp-frontend-dataset-controller.nomad

```diff
@@ -16,7 +16,7 @@ job "dp-frontend-dataset-controller" {
       value     = "web"
     }
 
-    task "dp-frontend-dataset-controller" {
+    task "dp-frontend-dataset-controller-web" {
```

Comment:
might be worth double checking with team C that is safe to change this without needing a corresponding change from them.